packages/nuxt-locokit/src/module.ts

```diff
@@ -442,34 +397,100 @@ export default defineNuxtModule<ModuleOptions>({
       /**
        * Register auth pages
        */
+      console.log('auth enabled ?', submodules.auth.enabled)
       if (submodules.auth.enabled) {
+        console.log('[nuxt-locokit] registering auth routes...')
         const prefix = submodules.auth.prefix
-        pages.push(...getAuthPages(prefix, pagesDir))
+        pages.push(
```

Comment:
Why this changed?

Reply:
Why keep `packages/nuxt-locokit/src/runtime/routes.ts`, then?

Reply:
Mmm, this has changed because I encounter (a lot) issues when building the nuxt module.
I discover that some paths are not really working, layouts are causing issues, etc.
I ended up to push all pages in the module files, but you're right, I remove the `routes.ts` file.